openreview/conference/builder.py

```diff
@@ -997,10 +1016,14 @@ def get_readers(self, conference, number):
         if conference.use_area_chairs:
             readers.append(conference.get_area_chairs_id(number = number))
 
-        if self.release_to_reviewers:
+        if self.release_to_reviewers is ReviewStage.Readers.REVIEWERS:
+            readers.append(conference.get_reviewers_id())
+        if self.release_to_reviewers is ReviewStage.Readers.REVIEWERS_ASSIGNED:
```

Comment:
Can we use elif instead of if for the 3 subsequent ifs ? Since, self.release_to_reviewers can only be 1 value out of these 4, it makes sense to not check all conditions as soon as 1 is met.

Reply:
I refactored to avoid the if else.